packages/jest-cli/src/cli/args.js

```diff
@@ -261,6 +261,12 @@ const options = {
       'Will run all tests affected by file changes in the last commit made.',
     type: 'boolean',
   },
+  forceExitAfterTestRun: {
+    default: false,
+    description:
+      'After run all tests that force exit.',
```

Comment:
'Force Jest to exit after all tests have completed running. This is useful when resources set up by test code cannot be adequately cleaned up.'